### DISCUSSION TITLE
Use date-fns to display absolute minutes

### DISCUSSION THREAD

**Discussion by niqwithq** [js-fns/date-fns#3464] [Q&A] [answered]
Hi, 

on my blog, I'm displaying reading time in minutes for each of my blog posts. Is there a way to show this information (= the minutes) as a `<time>` element, using date-fns?

Basically, this [stackoverflow post](https://stackoverflow.com/questions/31559103/html-time-tag-for-duration) summarizes my situation perfectly.

**Comment by maximtop** [comment, marked as answer]
In date-fns, you can handle this as follows:

```javascript
  const duration = { minutes: 13, seconds: 7 };
  
  // Refer to https://date-fns.org/v2.30.0/docs/formatISODuration for documentation
  const isoDurationStr = formatISODuration(duration);
  console.log(isoDurationStr); // Output: P0Y0M0DT0H13M7S - a valid value for the datetime attribute duration
  
  // Refer to https://date-fns.org/v2.30.0/docs/formatDuration for documentation
  const formattedDuration = formatDuration(duration);
  console.log(formattedDuration); // Output: 13 minutes 7 seconds - you can use whatever format suits you
 ```

Here's a CodeSandbox link for this example: https://codesandbox.io/s/date-fns-duration-format-pg2z5k?file=/src/App.js:119-167

**Reply by niqwithq** [reply]
Hi @maximtop ,
Beautiful. Thank you for the simple examples to achieve this. Works like a charm ✨ .

All the best
